bin/citadel

```diff
@@ -0,0 +1,276 @@
+if [[ "$command" = "reboot" ]]; then
+  sudo $CITADEL_ROOT/scripts/stop || true
+  sudo reboot
+  exit
+fi
+
+# Shutdown the system
+if [[ "$command" = "shutdown" ]]; then
+  sudo $CITADEL_ROOT/scripts/stop || true
+  sudo shutdown now
+  exit
+fi
+
+# App commands
+if [[ "$command" = "app" ]]; then
+  shift
+  sudo $CITADEL_ROOT/scripts/app $@
+  exit
+fi
+
+# Edit common app configuration files
+if [[ "$command" = "configure" ]]; then
+  if [ -z ${2+x} ]; then
+    echo "Specify an app or service to configure."
+    echo "Usage: \`$CLI_NAME configure <service>\`"
+    exit 1
+  fi
+
+  POSITIONAL_ARGS=()
+
+  persist=false
+
+  while [[ $# -gt 0 ]]; do
+    case $1 in
+    --persist)
+      persist=true
+      shift # past argument
+      ;;
+    -* | --*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      POSITIONAL_ARGS+=("$1") # save positional arg
+      shift                   # past argument
+      ;;
+    esac
+  done
+
+  set -- "${POSITIONAL_ARGS[@]}" # restore positional parameters
+
+  # These service and app configs are already persisted
+  # TODO: add more apps
+
+  if [[ "$2" = "nextcloud" ]]; then
+    edit_file --priviledged $CITADEL_ROOT/app-data/nextcloud/data/nextcloud/config/config.php
+    exit
+  fi
+
+  if [[ "$2" = "nginx" ]]; then
+    edit_file $CITADEL_ROOT/nginx/nginx.conf
+    echo "To apply the changes, restart Nginx service with \`docker restart nginx\`."
+    exit
+  fi
+
+  if $persist; then
+    echo "NOTE: As of now persisted config changes will not be kept when updating Citadel."
+  else
+    echo "NOTE: Some changes to this configuration file may be overwritten the next time you start Citadel."
+    echo "To persist changes run the command again with \`$CLI_NAME configure <service> --persist\`"
+  fi
+
+  read -p "Continue? [Y/n] " should_continue
+  echo
+  if [[ $should_continue =~ [Nn]$ ]]; then
+    exit
+  fi
+
+  # Service and app configs below may be overwritten
+  # TODO: check which implementation is running
+  # and do "bitcoin" / "lightning" / "electrum"
+
+  if [[ "$2" = "bitcoin" ]]; then
+    if $persist; then
+      edit_file $CITADEL_ROOT/templates/bitcoin-sample.conf
+      echo "To apply the changes, restart Citadel with \`$CLI_NAME restart\`."
+    else
+      edit_file $CITADEL_ROOT/bitcoin/bitcoin.conf
+      echo "To apply the changes, restart Bitcoin service with \`docker restart bitcoin\`."
+    fi
+    exit
+  fi
+
+  if [[ "$2" = "lnd" ]]; then
+    if $persist; then
+      edit_file $CITADEL_ROOT/templates/lnd-sample.conf
+      echo "To apply the changes, restart Citadel with \`$CLI_NAME restart\`."
+    else
+      edit_file $CITADEL_ROOT/lnd/lnd.conf
+      echo "To apply the changes, restart LND service with \`docker restart lightning\`."
+    fi
+    exit
+  fi
+
+  if [[ "$2" = "electrs" ]]; then
+    edit_file $CITADEL_ROOT/templates/electrs-sample.toml
+    echo "To apply the changes, restart Citadel with \`$CLI_NAME restart\`."
+    exit
+  fi
```

Comment:
To prevent the log message regarding persistent changes, please move this up in the file to the nginx config